example_projects/transfer/nlb_observability_stack/network.tf

```diff
@@ -0,0 +1,7 @@
+resource "doublecloud_network" "nlb-network" {
```

Comment:
NIT
guess, for this example will be better to setup BYOC and VPC peering

Reply:
discussed with @laskoviymishka , for now we will keep it without byoc :)